test/dist_autograd_test.py

```diff
@@ -78,6 +79,23 @@ def my_rref_add(rref_t1, t2):
     ret = torch.add(rref_t1.local_value(), t2)
     return ret
 
+@torch.jit.script
+def my_script_add(t1, t2):
+    return torch.add(t1, t2)
+
```

Comment:
Is it possible to write a torchscript function that throws an exception and validate that the client receives the appropriate exception? For example the body of the function could call torch operators with invalid args.